src/page-multi-facility/FacilityInputForm.tsx

```diff
@@ -16,13 +16,18 @@ import LocaleInformationSection from "./LocaleInformationSection";
 const FacilityInputFormDiv = styled.div`
   display: flex;
   flex-direction: row;
-  justify-content: space-between;
 `;
 const LeftColumn = styled.div`
   margin: 20px;
+  width: 45%;
 `;
 const RightColumn = styled.div`
   margin: 20px;
+  width: 55%;
```

Comment:
These numbers were chosen because when I compared the visual output to the current Impact Dashboard they looks almost identical